src/qt/bitcoingui.cpp

```diff
@@ -748,6 +748,15 @@ void BitcoinGUI::setNetworkActive(bool networkActive)
     updateNetworkState();
 }
 
+void BitcoinGUI::updateHeadersSyncProgressLabel()
+{
+    int64_t headersTipTime = clientModel->getHeaderTipTime();
+    int headersTipHeight = clientModel->getHeaderTipHeight();
+    int estHeadersLeft = (GetTime() - headersTipTime)/600;
```

Comment:
Nit: I know I initially put it there, but please change it to Consensus::Params::nPowTargetTimespan, as sipa requested.